paasta_tools/kubernetes_tools.py

```diff
@@ -724,6 +712,16 @@ def get_sidecar_containers(
         service_namespace_config: ServiceNamespaceConfig,
         hacheck_sidecar_volumes: Sequence[DockerVolume],
     ) -> Sequence[V1Container]:
+        return self.get_hacheck_sidecar_container(
+            system_paasta_config, service_namespace_config, hacheck_sidecar_volumes,
+        ) + self.get_uwsgi_exporter_sidecar_container(system_paasta_config)
```

Comment:
nit, but I find returning 1-element lists a little unexpected

Reply:
Yeah, I guess that's weird. It just saves me from having to do something like 
```
hacheck_sidecar = self.get_hacheck_sidecar_container(...)
if hacheck_sidecar is not None:
    sidecars.append(hacheck_sidecar)
```
maybe there's a better one-liner for that.